ee/tabby-ui/app/(dashboard)/page.tsx

```diff
@@ -1,24 +1,17 @@
 'use client'
 
-import { buttonVariants } from '@/components/ui/button'
-import {
-  Dialog,
-  DialogContent,
-  DialogDescription,
-  DialogFooter,
-  DialogHeader,
-  DialogTitle
-} from '@/components/ui/dialog'
-import { IconSlack } from '@/components/ui/icons'
-import { Separator } from '@/components/ui/separator'
-import { useHealth } from '@/lib/hooks/use-health'
-import { PropsWithChildren, useEffect, useState } from 'react'
+import {buttonVariants} from '@/components/ui/button'
+import {Dialog, DialogContent, DialogDescription, DialogFooter, DialogHeader, DialogTitle} from '@/components/ui/dialog'
+import {IconSlack} from '@/components/ui/icons'
+import {Separator} from '@/components/ui/separator'
+import {useHealth} from '@/lib/hooks/use-health'
+import {PropsWithChildren, useEffect, useState} from 'react'
 import WorkerCard from './components/worker-card'
-import { useWorkers } from '@/lib/hooks/use-workers'
-import { WorkerKind } from '@/lib/gql/generates/graphql'
-import { useGraphQL } from '@/lib/hooks/use-graphql'
-import { getRegistrationTokenDocument } from '@/lib/gql/request-documents'
-import { CopyButton } from '@/components/copy-button'
+import {useWorkers} from '@/lib/hooks/use-workers'
+import {WorkerKind} from '@/lib/gql/generates/graphql'
+import {useGraphQL} from '@/lib/hooks/use-graphql'
+import {getRegistrationTokenDocument} from '@/lib/gql/request-documents'
+import {CopyButton} from '@/components/copy-button'
```

Comment:
Don't mix formatting changes. (similar comments on other ts files with import style changed)

Probably we should maintain a .editorconfig to avoid situations like this in future.

Reply:
Yeah otherwise IntelliJ IDEs will just format everything according to their code style as you can see. They do prefer keeping everything as they want if not told otherwise. I'd recommend adding prettier or so.

Reply:
To be clear this is not prettier, this is just IntelliJ formatting every import that it touches, which it does if there aren't prettier or a properly configured editorconfig, which is exactly why it was good to have one. I didn't change it on purpose, but there are no defaults here, so the IDE used its own defaults.

Reply:
Should be better now after I manually adjusted some Rust Rover configuration